### ISSUE TITLE
How to do evaluation of conformer onnx model?

### ISSUE THREAD

**Issue by neso613** [wenet-e2e/wenet#1458]
Hi team

I have converted [Conformer model pre-trained on aishell](https://github.com/wenet-e2e/wenet/blob/main/docs/pretrained_models.md) to ONNX successfully. 

Now, I want to check the accuracy of ONNX model but dont find any doc for this. What I find myself is [recognize_onnx.py](https://github.com/wenet-e2e/wenet/blob/main/wenet/bin/recognize_onnx.py) is used but preprocessing steps are missing?

**Comment by xingchensong** [comment]
c++ runtime for onnx: https://github.com/wenet-e2e/wenet/tree/main/runtime/onnxruntime

**Comment by xingchensong** [comment]
If you have gflag/glog/openfst installed by root user, uninstall them and try again

**Comment by neso613** [comment]
> gflag/glog/openfst

How can I remove it...I dont find any proper way to doing it. Will you pls suggest some options.
